File: pytools/data/data.code.php.server/ReqSceneEnterFly.php
<?php
namespace protocol;

use network\Packet;


class ReqSceneEnterFly
{
	private $mapId = 0;


	public function __construct(&$packet)
	{
		$this->decode($packet);
	}

	public function decode(&$packet)
	{
		$this->mapId = $packet->readU32();
	}


	public function getMapId()
	{
		return $this->mapId;
	}

}

File: pytools/data/data.code.java.server/ReqSceneEnterFly.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class ReqSceneEnterFly
{
	private int map_id;


	public ReqSceneEnterFly(Packet packet)
	{
		map_id = packet.readInt();
	}



	public long getMapId()
	{
		return PacketUtil.readUInt(this.map_id);
	}

}
